### PR TITLE
fix(environment-variables): Clarify creation of .env file

### PR DESCRIPTION
**Pull request by janpio**
Reported via https://github.com/prisma/prisma/issues/19453

### CODE REVIEW

File: content/300-guides/125-development-environment/050-environment-variables/index.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ Taking the `TEMP` environment variable as an example, one can query its value to
 
 Prisma always reads environment variables from the system's environment.
 
-When you install Prisma for the first time it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in there are put into the system's environment, where Prisma can read it and use it.
+When you initially create your Prisma setup with `prisma init`, it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in there are put into the system's environment, where Prisma can read it and use it.
```
Comment (Jolg42) [inline content/300-guides/125-development-environment/050-environment-variables/index.mdx]:
```suggestion
When you initially create your Prisma setup with `prisma init`, it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in there are added to the [`process.env` object](https://nodejs.org/api/process.html#processenv), where Prisma can read it and use it.
```

"system's environment" for me would mean that anything on that computer can read these env, in fact we only load them into the current `process.env` of the current Node instance.

Comment (jharrell) [inline content/300-guides/125-development-environment/050-environment-variables/index.mdx]:
```suggestion
When you initially create your Prisma setup with `prisma init`, it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use the Prisma CLI or Prisma Client, the `.env` file and the variables defined in it are put into the system's environment, where Prisma can read it and use it.
```

Comment (janpio) [reply content/300-guides/125-development-environment/050-environment-variables/index.mdx]: (outdated now, have to create again so can be accepted)

Comment (janpio) [reply content/300-guides/125-development-environment/050-environment-variables/index.mdx]: ping @jharrell

File: content/300-guides/125-development-environment/050-environment-variables/index.mdx [hunk 2]
```diff
@@ -25,7 +25,7 @@ Taking the `TEMP` environment variable as an example, one can query its value to
 
 Prisma always reads environment variables from the system's environment.
 
-When you install Prisma for the first time it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in there are put into the system's environment, where Prisma can read it and use it.
+When you initially create your Prisma setup with `prisma init`, it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in there are added to the [`process.env` object](https://nodejs.org/api/process.html#processenv), where Prisma can read it and use it.
```
Comment (ruheni) [inline content/300-guides/125-development-environment/050-environment-variables/index.mdx]:
```suggestion
When you initialize Prisma in your project with `prisma init`, it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in it are added to the [`process.env` object](https://nodejs.org/api/process.html#processenv), where Prisma can read it and use it.
```

File: content/300-guides/125-development-environment/050-environment-variables/index.mdx (hunk 1 above)
Comment (jharrell) [reply content/300-guides/125-development-environment/050-environment-variables/index.mdx]: Thanks, updated version looks good.

**Review by jharrell — APPROVED** [review]

**Pull request merged**